base/doc/ltnews39.tex

```diff
@@ -157,7 +157,7 @@ \section{Introduction}
 we expect to gradually lift these restrictions and eventually provide
 the full functionality as part of the core distribution, rather than
 through \texttt{latex-lab} modules.}  A sample collection of such
-documents ranging from classical text such as the Bible to recent
+documents ranging from classical texts, such as the Bible, to recent
 technical papers submitted to arXiv.org. These can be found at
```

Comment:
There is either a verb missing (A sample collection ... has been created). Or there is a period too much (A sample collection  ... can be found ...)